src/catalogue/items/CatalogueItemsDetailsPanel.component.tsx

```diff
@@ -197,7 +197,9 @@ function CatalogueItemsDetailsPanel(props: CatalogueItemsDetailsPanelProps) {
                     property.unit ? `(${property.unit})` : ''
                   }`}</Typography>
                   <Typography color="text.secondary">
-                    {String(property.value)}
+                    {String(property.value).trim() !== 'null'
+                      ? String(property.value)
+                      : 'None'}
```

Comment:
```suggestion
                    {property.value !== null ? String(property.value) : 'None'}
```
The api just returns null here doesn't it? This is also the only one not in the coverage, not sure its worth it though unless its simple as the rest are.